src/lib/storage/fake_device_factory.rb

```diff
@@ -86,8 +84,9 @@ def initialize(devicegraph)
         super(devicegraph)
         @partitions     = {}
         @disks          = Set.new
-        @first_free_cyl = {}
-        @cyl_count      = {}
+        @free_blob      = nil
+        @free_regions   = []
+        @mbr_gap        = nil
```

Comment:
Actually, yes (but for a bug I just noticed).

But this is in fact something I wanted to ask you about. As something like
`@mbr_gap        = nil`
looks pretty weird, or at least highly un-ruby, to me.